airflow/kubernetes/kube_client.py

```diff
@@ -58,6 +58,11 @@ def _get_client_with_patched_configuration(cfg: Optional[Configuration]) -> clie
         else:
             return client.CoreV1Api()
 
+    def _disable_verity_ssl() -> None:
+        configuration = client.Configuration()
+        configuration.verify_ssl = False
+        client.Configuration.set_default(configuration)
```

Comment:
I am not sure how this works -- can you explain please since we don't use the client you are using over here

Reply:
when use `in_cluster` mode, Python k8s client will load incluster config, but certificate bundle `/var/run/secrets/kubernetes.io/serviceaccount/ca.crt` is invalid on alicloud.
set `verify_ssl` can skip verifying SSL certificate
https://github.com/kubernetes-client/python/blob/master/kubernetes/client/configuration.py#L130

Reply:
relate issue: https://github.com/kubernetes-client/python/issues/490
and cleanup pod error:
```Max retries exceeded with url: /api/v1/namespaces/airflow/pods?continue=None&limit=500 (Caused by SSLError(SSLError("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')],)",),))```

Reply:
Is it possible to trace the error with invalid certificate on Alicloud somewhere?

Reply:
@mik-laj I only find out related issue https://github.com/getsentry/sentry-kubernetes/pull/38
It seems that internal implementation for alicloud